build/Targets/References.targets

```diff
@@ -139,7 +148,7 @@
         <Reference Include="Microsoft.VisualStudio.ComponentModelHost">
           <HintPath>$(DevEnvDir)\PublicAssemblies\Microsoft.VisualStudio.ComponentModelHost.dll</HintPath>
         </Reference>
-	<!--
+        <!--
```

Comment:
They weren't intentional (I must have done them without really thinking about it) - but I looked at them all and I'm okay with the changes. Had "ignore whitespace" turned on, so I didn't see them in my diff, will make sure that future commits separate them.